Rivet/Functions/Operations/Add-Description.ps1

```diff
@@ -30,25 +45,44 @@ function Add-Description
         # The value for the MS_Description extended property.
         $Description,
 
+        [Parameter(ParameterSetName='ForSchema')]

```

Comment:
I think you can omit all these `[Parameter]` attributes since there are no other parameter sets betsides these four.